lib/saml_idp/incoming_metadata.rb

```diff
@@ -60,17 +60,6 @@ def encryption_certificate
     end
     hashable :encryption_certificate
 
-    def single_logout_services
```

Comment:
this is probably fine to leave in right? Looks like it's for incoming metadata and not for the metadata we publish?